server/main.lua

```diff
@@ -0,0 +1,66 @@
+---@class InsertVehicleEntityQuery
+---@field license string
+---@field citizenId string
+---@field model string
+---@field plate string
+---@field mods string
+
+--- Creates a Vehicle DB Entity
+---@param query InsertVehicleEntityQuery Insert Vehicle Entity Request
+local function createEntity(query)
+    MySQL.insert('INSERT INTO player_vehicles (license, citizenid, vehicle, hash, mods, plate, state) VALUES (?,?,?,?,?,?,?)', {
+        query.license,
+        query.citizenId,
+        query.model,
+        joaat(query.model),
+        query.mods or '{}',
+        query.plate,
+        0
+    })
+end
```

Comment:
Should we overload this to take in PlayerData and vehicle as params? This is a pretty low level call right now. I think for most use cases, we could derive the needed fields based on having the vehicle.

Reply:
I've just pushed my local with this change in, but I feel like a misunderstood slightly. Would you be able to take a look to see if it is along the same lines. I will trim it down so that we can just do the vehicle derivations later.